setup.py

```diff
@@ -28,6 +29,7 @@
         "Framework :: Sphinx :: Theme",
     ],
     install_requires=[
+        "furo @ git+https://github.com/pradyunsg/furo.git@48c0bf2aa983fb1ccd79b4167fe85387249afedf",
```

Comment:
Should we use a release tag rather than a specific commit?

Reply:
They have not yet released Furo with a version that works with theme inheritance. I had to make a patch to get Furo working, so we are consumed from a newer version than the last release on PyPI.

We'll change this to a formal release the next time they make one. (I watch the repository so I will get a notification)